test/lapack_wrappers.hh

```diff
@@ -115,7 +115,7 @@ double lapack_lantr( char const *norm, char const *uplo, char const *diag,
                      double *work );
 
 // -----------------------------------------------------------------------------
-template< typename TA, typename TB > inline
+template <typename TA, typename TB> inline
```

Comment:
The `inline` keyword was kep here but not above, why?

Reply:
It could be removed. `inline` isn't needed on `template` functions.